woocommerce/wc-admin
# Build: Update packages

I used `updtr` to locally update all packages to their latest versions, and fixed the few issues that came out of the upgrades. The exception is `docsify-cli`, which is now pinned to [a commit SHA](https://github.com/docsifyjs/docsify-cli/commit/5df389962a0eb69ac02701b297b77b2fbb85ec28) to fix a vulnerability in a dependency, which hasn't been published to npm.

- Update snapshot with new Jest format for child components
- Add a few packages that ended up being missing due to upgrades splitting packages
- Updates the engine node & npm versions to match gutenberg #433 
- Regenerate docs (to test the build process, but also we're outdated)

**To test**

- Run build or start & check that no functionality has changed
- Check that tests still pass

We could check out https://renovatebot.com/ for automatically doing this in the future?
